paasta_tools/setup_kubernetes_cr.py

```diff
@@ -240,6 +240,13 @@ def get_dashboard_base_url(kind: str, cluster: str) -> Optional[str]:
     return None
 
 
+# FLINK-4287: This should be more generalized later to include owners of other kinds
+def get_cr_owner(kind: str) -> Optional[str]:
+    if kind.lower() == "flink":
```

Comment:
would you mind adding kafkacluster, cassandracluster and nrtsearchservice?

Reply:
Do you have the owner names? I am not sure about the names here.

Reply:
My mapping of ownership is:
```
jenkins (cid)
jolt (cid)
paasta (compute_infra_platform_experience)
paasta-cassandraclusters (database-reliability-engineering-nosql)
paasta-flinks (stream-processing)
paasta-kafkaclusters (kafka-operator)
paasta-monkrelays (streaming-platform)
paasta-nrtsearchservices (ranking_platform)
paasta-scyllaclusters (cid)
paasta-spark (core_ml)
tron (compute_infra_platform_experience)
rest/system_stuff (compute_infra_core)
```

Reply:
so `cassandracluster -> database-reliability-engineering-nosql`, `flink -> stream-processing`, `kafkacluster -> kafka-operator`, , `monkrelay -> streaming-platform` (no CRD yet, i think but cant hurt), `nrtsearchservice -> ranking_platform` and `scyllacluster -> cid`

Reply:
Done

Reply:
Are we sure kafkacluster owner would be `kafka-operator`?